docs/softwares.md

```
@@ -437,186 +437,163 @@ verifybamid2 estimates contamination in samples by simultaneously estimating gen
 
-## somalier
+## somalier_matched
 
-## [somalier_combine_fam](https://github.com/brentp/somalier)
+## [somalier_matched_combine_fam](https://github.com/brentp/somalier)
 Combines individual tumor and normal pedigree files into a single master .fam file for batch analysis.
 
 ### :snake: Rule
 
-#SNAKEMAKE_RULE_SOURCE__somalier__somalier_combine_fam#
+#SNAKEMAKE_RULE_SOURCE__somalier_matched__somalier_matched_combine_fam#
```

markdownlint-cli2 (0.20.0)

No space inside hashes on closed atx style heading

(MD020, no-missing-space-closed-atx)